AMM/project/scripts/swap-exact-input/tests/harness.rs

```diff
@@ -0,0 +1,301 @@
+use fuels::prelude::*;
```

Comment:
The following 2 scripts contain a `harness.rs` which has the setup and test cases. Splitting into `revert.rs` and `success.rs`, similar to the atomic script, should be fine